src/main/java/com/rybka/config/FileUtils.java

```diff
@@ -0,0 +1,19 @@
+package com.rybka.config;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+
+public class FileUtils {
+
+    public static final String FILE_NAME_PATTERN = "yyyy-MM-dd_hh-mm-ss";
+    public static final String CSV_SUFFIX = ".csv";
+    public static final String JSON_SUFFIX = ".json";
+
+    public static String generateFileName(String fileSuffix) {
+        Date date = Calendar.getInstance().getTime();
+        DateFormat dateFormat = new SimpleDateFormat(FILE_NAME_PATTERN);
```

Comment:
This field can be private static. No need to create dateFormat each time on method call.

Reply:
We should consider the same for Date object. 
Personally, I prefer LocalDate/LocalDateTime from Java 8.

Reply:
fixed